vulcanai/models/basenetwork.py

```diff
@@ -680,13 +680,15 @@ def _validate(self, val_loader):
 
         return validation_loss, validation_accuracy
 
-    def run_test(self, data_loader, plot=False, save_path=None):
+    def run_test(self, data_loader, plot=False, save_path=None, pos_label=1):
```

Comment:
the one thing I'm worried about is what if the user is predicting multiple positive labels in one instance?

Reply:
I'm not sure I understand.... this is only for the binary case. (which reminds me that I need to add that to the documentation)

Reply:
so the pos_label is only used for the binary case? let's just make sure that is clear in the docs, thanks!

Reply:
yeah, I'll do that. does everything else look ok? can you take a look at the diff for the auc function?

Reply:
yup ill take a look!